groovy/postInit/chemistry/inorganic_chemistry/elements/f_block/actinides/UraniumChain.groovy

```diff
@@ -28,44 +28,86 @@ ELECTROMAGNETIC_SEPARATOR = recipemap('electromagnetic_separator')
 PSA = recipemap('pressure_swing_adsorption')
 CHEMICAL_BATH = recipemap('chemical_bath')
 ROTARY_KILN = recipemap('rotary_kiln')
+VACUUM_CHAMBER = recipemap('vacuum_chamber')
+ION_EXCHANGE = recipemap('ion_exchange_column')
 
-MIXER.recipeBuilder()
+// Beneficiation
+
+    // Carnotite
+
+    MIXER.recipeBuilder()
+        .inputs(ore('dustCarnotite'))
         .fluidInputs(fluid('hot_sulfuric_acid') * 2000)
         .fluidInputs(fluid('distilled_water') * 4000)
-        .inputs(ore('dustCarnotite'))
+        .fluidOutputs(fluid('crude_carnotite_leach') * 6000)
+        .fluidOutputs(fluid('helium') * 5)
+        .EUt(120)
+        .duration(300)
+        .buildAndRegister()
+
+    VACUUM_CHAMBER.recipeBuilder()
+        .notConsumable(metaitem('item_filter'))
+        .fluidInputs(fluid('crude_carnotite_leach') * 6000)
+        .chancedOutput(metaitem('dustGraniteTailings'), 5000, 0)
         .fluidOutputs(fluid('carnotite_leach') * 6000)
         .EUt(120)
         .duration(300)
         .buildAndRegister()
 
-MIXER.recipeBuilder()
-        .fluidInputs(fluid('carnotite_leach') * 6000)
+    MIXER.recipeBuilder()
         .inputs(ore('dustAnyPurityIron'))
         .inputs(ore('dustCalcite') * 3)
+        .fluidInputs(fluid('carnotite_leach') * 6000)
         .fluidOutputs(fluid('reduced_carnotite_leach') * 6000)
         .EUt(120)
         .duration(300)
         .buildAndRegister()
 
-MIXER.recipeBuilder()
-        .fluidInputs(fluid('kerosene') * 900)
-        .fluidInputs(fluid('tributyl_phosphate') * 40)
-        .fluidInputs(fluid('di_two_ethylhexyl_phosphoric_acid') * 60)
-        .fluidOutputs(fluid('uranium_vanadium_extraction_mixture') * 1000)
+    // Pitchblende
+
+    // 3 UO2 + NaClO3 -> UO2/2+ + 3H2O + NaCl
+    MIXER.recipeBuilder()
+        .inputs(ore('dustPitchblende') * 3)
+        .inputs(metaitem('gregtechfoodoption:sodium_chlorate_dust') * 4) // this is intentionally less than a mole to account for U(VI) in the ore
+        .fluidInputs(fluid('diluted_sulfuric_acid') * 6000)
+        .fluidOutputs(fluid('crude_pitchblende_leach') * 5400)
+        .fluidOutputs(fluid('helium') * 5)
```

Comment:
would it make more sense to output 6kL crude_pitchblende_leach since you are inputting 5.4kL diluted sulfuric acid?

Reply:
no, water 2400L of water is produced and 3000 comes from the acid